src/main/java/com/epam/gym_app_stats_mq/stat/statMongoDb/StatModelMongoDb.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.gym_app_stats_mq.stat.statMongoDb;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Document(collection = "trainer_stats")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class StatModelMongoDb implements Serializable {
+    private ObjectId id;
+    @Indexed(unique = true)

```

Comment:
This index works only for one field, but you need Trainer First Name & Trainer Last Name